### DISCUSSION TITLE
Add timezone info into time

### DISCUSSION THREAD

**Discussion by zakuro9715** [vlang/v#11740] [Ideas]
Current time donesn't have timezone info. But I think time should have timezone info
 
e.g.: In golang, time has timezone as Location. https://golang.org/src/time/time.go#L127

**Comment by dumblob** [comment]
This seems to be a duplicate of https://github.com/vlang/v/issues/3654 .

Proper support for timezone will be definitely needed. There are though many issues with that (and golang's API ignores some of them - especially when adding/subtracting times and/or time differences) - see https://github.com/vlang/v/issues/2586#issuecomment-640611953 .

My advice would be to look at how [Dao time](https://github.com/daokoder/dao-modules/blob/master/time/dao_time.c#L1920-L2101 ) implementation looks like.
